README.md

```diff
@@ -66,11 +90,11 @@ Which of the following statements will work, assuming jQuery is loaded?
 
 Select all that apply:
 ```
-[] `$(".post").css("background", "peachpuff")`
+[X] `$(".post").css("background", "peachpuff")`
 [] `$(".post").innerHTML`
-[] `$(".post").html()`
-[] `document.querySelectorAll(".post")[0].innerHTML`
-[] `document.querySelectorAll(".post").innerHTML`
+[X] `$(".post").html()`
+[X] `document.querySelectorAll(".post")[0].innerHTML`
+[X] `document.querySelectorAll(".post").innerHTML`
```

Comment:
This last option attempts to call "InnerHTML" on an array of retrieved elements.

:+1:  The rest are correct.